client/src/layout/Router/Router.tsx

```diff
@@ -17,6 +20,8 @@ export const Router = (_props: RouterProps) => {
                     <Route path={ROUTES.MATCH_EXPERIENCES} element={<MatchExperiencesCatalogScreen mode="all" />} />
                     <Route path={`${ROUTES.MATCH_EXPERIENCES}/:id`} element={<MatchExperienceDetailsScreen />} />
                     <Route path={ROUTES.MY_EXPERIENCES} element={<MatchExperiencesCatalogScreen mode="my" />} />
+                    <Route path={ROUTES.PACKAGES} element={<PackagesScreen/>} />
```

Comment:
```suggestion
                    <Route path={`${ROUTES.PACKAGES}/results`}element={<PackagesScreen/>} />
```
just to make it clearer